ChemCheck/upload/views.py

```diff
@@ -56,12 +57,16 @@ def ck2yaml(request, pk):
                         permissive = True,
                         )
     except Exception as e:
+        error = os.path.join(os.path.split(input_file)[0], 'error.txt')
+        content = open(error, "r").read()
+        conversion_log += str(content)
         conversion_log += str(e)                      
         error_message = traceback.format_exc()
         conversion_log += error_message
         mechanism.ct_conversion_errors = error_message
         mechanism.ct_mechanism_file = None
         mechanism.save()
+        open (error, "w").close()
```

Comment:
What does this line do?

Reply:
Hi @bryanwweber 
This line is to delete the content in 'error.txt' after every conversion. Because error.txt is a file to record logging messages, it will record messages after every conversion, but I just want to render the logging messages generated from one conversion instead all of the messges from previous conversions (if I run conversion more than once). 

Without this line, there will be several duplicate logging messages depends on how much time you run the conversion (hit the 'convert to yaml' button). 

This maybe not a clever way to solve duplicated logging messages, so please let me know if you have a better idea.

Reply:
Why not just remove the file all together? https://stackoverflow.com/a/6996628/2449192 At some point, we will want to move this into a buffer instead of a file, but this is fine for now. Make sure to check if the file exists before you remove it.

If you're using a file on the disk, you also need to make sure it is unique per-conversion, I think. Otherwise, if two people try to make a conversion at the same time, they will try to write to the same file at the same time and that wouldn't be good.

Reply:
Think this is resolved. We moved where it happens. Conflicts should't occur (any more than they do for the CTI files) because each mechanism is converted in a different directory.